tracer/src/Datadog.Trace/SpanContextPropagator.cs

```diff
@@ -114,8 +121,12 @@ public void Inject<TCarrier>(SpanContext context, TCarrier carrier, Action<TCarr
             var parentId = ParseUInt64(carrier, getter, HttpHeaderNames.ParentId) ?? 0;
             var samplingPriority = (SamplingPriority?)ParseInt32(carrier, getter, HttpHeaderNames.SamplingPriority);
             var origin = ParseString(carrier, getter, HttpHeaderNames.Origin);
+            var datadogTags = ParseString(carrier, getter, HttpHeaderNames.DatadogTags);
 
-            return new SpanContext(traceId, parentId, samplingPriority, serviceName: null, origin);
+            return new SpanContext(traceId, parentId, samplingPriority, serviceName: null, origin)
+                   {
+                       DatadogTags = datadogTags
+                   };
```

Comment:
```suggestion
            return new SpanContext(traceId, parentId, samplingPriority, serviceName: null, origin)
            {
                DatadogTags = datadogTags
            };
```

Reply:
We must have different formatting settings 😅